From 5dec7401cb4fa67e2731f6e2b480959777b90d46 Mon Sep 17 00:00:00 2001
From: "Simental Magana, Marcos" <marcos.simental.magana@intel.com>
Date: Wed, 22 May 2019 09:50:46 -0500
Subject: [PATCH 1/2] Add Software application to the favorite list

this commit adds 'org.gnome.Software.desktop' launcher to the
favorite-apps list for org.gnome.shell preferences.

Fixes https://github.com/clearlinux/distribution/issues/773

Signed-off-by: Simental Magana, Marcos <marcos.simental.magana@intel.com>
---
 schemas/org.gnome.shell.gschema.override | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/schemas/org.gnome.shell.gschema.override b/schemas/org.gnome.shell.gschema.override
index f1008e6..992f367 100644
--- a/schemas/org.gnome.shell.gschema.override
+++ b/schemas/org.gnome.shell.gschema.override
@@ -1,3 +1,3 @@
 [org.gnome.shell]
 enabled-extensions=['dash-to-dock@micxgx.gmail.com', 'desktop-icons@csoriano', 'impatience@gfxmonk.net', 'user-theme@gnome-shell-extensions.gcampax.github.com']
-favorite-apps=['org.gnome.Terminal.desktop', 'org.gnome.Nautilus.desktop', 'firefox.desktop', 'evolution.desktop']
+favorite-apps=['org.gnome.Terminal.desktop', 'org.gnome.Nautilus.desktop', 'firefox.desktop', 'evolution.desktop', 'org.gnome.Software.desktop']
-- 
2.21.0

